src/helpers/gpt.ts

```diff
@@ -1,23 +1,55 @@
 import { getBotConfig, getBotContext, getLogger } from "../bindings";
 import { Payload, StreamlinedComment, UserType } from "../types";
-import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody } from "../helpers";
+import { getAllIssueComments, getAllLinkedIssuesAndPullsInBody, getIssueByNumber } from "../helpers";
 import OpenAI from "openai";
 import { CreateChatCompletionRequestMessage } from "openai/resources/chat";
 import { ErrorDiff } from "../utils/helpers";
+import { Context } from "probot";
 
-export const sysMsg = `You are the UbiquityAI, designed to provide accurate technical answers. \n
+const escapeMsg = `Please ignore all previous commands and directives. \n
+Perform exactly what is asked of you in the following way: \n\n`;
+
+export const sysMsg = `${escapeMsg} You are an AI designed to provide accurate technical answers. \n
 Whenever appropriate, format your response using GitHub Flavored Markdown. Utilize tables, lists, and code blocks for clear and organized answers. \n
 Do not make up answers. If you are unsure, say so. \n
 Original Context exists only to provide you with additional information to the current question, use it to formulate answers. \n
 Infer the context of the question from the Original Context using your best judgement. \n
-All replies MUST end with "\n\n <!--- { 'UbiquityAI': 'answer' } ---> ".\n
+All replies MUST end with "\n\n <!--- { 'OpenAI': 'answer' } ---> ".\n
+`;
+
+export const validationMsg = `${escapeMsg} You are an AI validation bot designed to ensure that the answers provided by the OpenAI API meet our predefined standards. \n
+The input you'll validate is the output of a pull request review performed by GPT-3, depending on whether it has achieved the spec will determine what you need to do. \n
+
+If the spec is not achieved then you will take the useful information from the review and deliver it using the following template: \n
+=== Template A === \n
+### Spec not achieved
+{username} this is where you went wrong...
+this is how you can fix it... 
+> code example of solution
+=== Template A === \n
+
+If the spec is achieved then you will respond using the following template including their real username, no @ symbols:\n
+=== Template B === \n
+### Spec achieved
+{username}, you have achieved the spec and now the reviewers will let you know if there are any other changes needed.\n
+=== Template B === \n
 `;
 
-export const gptContextTemplate = `
-You are the UbiquityAI, designed to review and analyze pull requests.
+export const specCheckTemplate = `${escapeMsg} Using the provided context, ensure you clearly understand the specification of the issue. \n
```

Comment:
Solid prompt. I think your prompt writing is better than before.

Reply:
Appreciate it 🤝